src/14_cal.py

```diff
@@ -6,19 +6,40 @@
 Write a program that accepts user input of the form
   `14_cal.py month [year]`
 and does the following:
- - If the user doesn't specify any input, your program should
-   print the calendar for the current month. The 'datetime'
-   module may be helpful for this.
- - If the user specifies one argument, assume they passed in a
-   month and render the calendar for that month of the current year.
- - If the user specifies two arguments, assume they passed in
-   both the month and the year. Render the calendar for that
-   month and year.
+ NONE => current month
+ - one arg => curren month
+ - month and year
  - Otherwise, print a usage statement to the terminal indicating
    the format that your program expects arguments to be given.
    Then exit the program.
 """
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+
+def program():
+    userInput = input("Please enter the month, then the year:")
```

Comment:
You might specify the data type of the month and year to avoid user error.  i.e., what happens when a user types "may"?